src/scripts/services/user-service.ts

```diff
@@ -146,7 +149,13 @@ export class OktaUserService {
           const users: User[] = [];
           return (
             maybeGroupOrClient
-              .listUsers()
+              .listUsers({
+                filter: listAll
+                  ? Object.keys(okta.UserStatus)
```

Comment:
**Issue:** this would benefit from a link to Okta's documentation explaining why adding this clause is necessary to list all users. My assumption is that this is because if this is not here, then Okta adds its own status clause.

Reply:
Yup i think you're right. 
![image](https://github.com/agiledigital-labs/oktagon/assets/124639768/9c43a7fb-2bd8-4714-893d-c1648ca40880)

Reply:
modified in commit: 7a96d0311eb5ea9e65b7a6114332f7c01b957ae4